PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please rename this file to `Scan Profile.xml`: https://github.com/pdq/PowerShell-Scanners/blob/master/Policies/Contributing.md#folder-structure

Reply:
I've renamed according to contributing guidelines, I'm new to Git though, should I do a new pull request?

Reply:
Nope. Additional commits made to the branch will get added to this PR., and it looks like they have accoding to de06d98